### PR TITLE
Adds admin logging to having vents siphon

### PR DESCRIPTION
**Pull request by Pepsilawn**
## About The Pull Request

This PR adds logging for when a vent is set to siphoning/pressurizing, which is an underutilized but deadly mechanic. Previously there was no way of telling through logs when somebody set a vent to siphon.
Changelog didn't feel necessary unless the average player enjoys looking at atmos logs, I can add one if needed.

## Why It's Good For The Game

When somebody starts setting all vents in the hallway or the Supermatter chamber to siphon which makes for a quick and hard to solve delam, it is good for staff to be able to investigate the situation.

### CODE REVIEW

File: code/modules/atmospherics/machinery/components/unary_devices/vent_pump.dm [hunk 1]
```diff
@@ -210,10 +210,12 @@
 	if("purge" in signal.data)
 		pressure_checks &= ~EXT_BOUND
 		pump_direction = SIPHONING
+		investigate_log(" pump direction was set to [pump_direction] by [key_name(signal_sender)]",INVESTIGATE_ATMOS)
```
Comment (dragomagol) [inline code/modules/atmospherics/machinery/components/unary_devices/vent_pump.dm:213]:
```suggestion
		investigate_log("pump direction was set to [pump_direction] by [key_name(signal_sender)]",INVESTIGATE_ATMOS)
```
These logs already start with a space, no need to add another

File: code/modules/atmospherics/machinery/components/unary_devices/vent_pump.dm [hunk 2]
```diff
@@ -210,10 +210,12 @@
...
 
 	if("stabilize" in signal.data)
 		pressure_checks |= EXT_BOUND
 		pump_direction = RELEASING
+		investigate_log(" pump direction was set to [pump_direction] by [key_name(signal_sender)]",INVESTIGATE_ATMOS)
```
Comment (dragomagol) [inline code/modules/atmospherics/machinery/components/unary_devices/vent_pump.dm:218]: Same here

**Comment by Fikou** [comment]
you dont want [s] for non security/secret prs

File: code/modules/atmospherics/machinery/components/unary_devices/vent_pump.dm (hunk 2 above)
Comment (Pepsilawn) [reply code/modules/atmospherics/machinery/components/unary_devices/vent_pump.dm:218]: sounds good

**Comment by Pandarsenic** [comment]
Remind me, is EMPs setting air alarms to siphon a thing, and if so does this handle that well?

File: code/modules/atmospherics/machinery/components/unary_devices/vent_pump.dm [hunk 3]
```diff
@@ -210,10 +210,12 @@
 	if("purge" in signal.data)
 		pressure_checks &= ~EXT_BOUND
 		pump_direction = SIPHONING
+		investigate_log("pump direction was set to [pump_direction] by [key_name(signal_sender)]",INVESTIGATE_ATMOS)
 
 	if("stabilize" in signal.data)
 		pressure_checks |= EXT_BOUND
 		pump_direction = RELEASING
+		investigate_log("pump direction was set to [pump_direction] by [key_name(signal_sender)]",INVESTIGATE_ATMOS)
```
Comment (LemonInTheDark) [inline code/modules/atmospherics/machinery/components/unary_devices/vent_pump.dm:218]:
```suggestion
		investigate_log("pump direction was set to [pump_direction] by [key_name(signal_sender)]", INVESTIGATE_ATMOS)

	if("stabilize" in signal.data)
		pressure_checks |= EXT_BOUND
		pump_direction = RELEASING
		investigate_log("pump direction was set to [pump_direction] by [key_name(signal_sender)]", INVESTIGATE_ATMOS)
```

if you wanna be real based add a setter for this to hold the duplicate behavior. Don't need to tho

Comment (Pepsilawn) [reply code/modules/atmospherics/machinery/components/unary_devices/vent_pump.dm:218]: I don't really have experience doing that sorry

**Comment by Pepsilawn** [comment]
> Remind me, is EMPs setting air alarms to siphon a thing, and if so does this handle that well?

EMP or a mob pulsing the air alarm's Panic Siphon wire is already properly logged.

**Pull request merged**